beacon_node/execution_layer/src/test_utils/execution_block_generator.rs

```diff
@@ -18,12 +18,16 @@ use std::sync::Arc;
 use tree_hash::TreeHash;
 use tree_hash_derive::TreeHash;
 use types::{
-    BlobSidecar, ChainSpec, EthSpec, ExecutionBlockHash, ExecutionPayload, ExecutionPayloadCapella,
-    ExecutionPayloadDeneb, ExecutionPayloadHeader, ExecutionPayloadMerge, ForkName, Hash256,
-    Transactions, Uint256,
+    Blob, ChainSpec, EthSpec, EthSpecId, ExecutionBlockHash, ExecutionPayload,
+    ExecutionPayloadCapella, ExecutionPayloadDeneb, ExecutionPayloadHeader, ExecutionPayloadMerge,
+    ForkName, Hash256, Transactions, Uint256,
 };
 
 use super::DEFAULT_TERMINAL_BLOCK;
+use ssz::Decode;
+
+const TEST_BLOB_BUNDLE_MAINNET: &[u8] = include_bytes!("fixtures/mainnet/test_blobs_bundle.ssz");
```

Comment:
Just a minor concern.
It seems like you generated 1 random `BlobBundle` and have written it to file. 
Do any of our tests assume different blobs within a block? I can't find any but just wondering if this would lead to tests passing where it shouldn't. 

Perhaps we store can store 6 different blobs in the written bundle? The size increase of 1mb shouldn't make much difference no? This wouldn't be an increase in the lighthouse binary size either iiuc.

Reply:
Yeah I thought about a few options before going down this route:
1.  pre-generating 6 blobs in a bundle, this way we only need one pre-generated bundle, but we may unnecessarily decode 6 blobs every time we use the test blobs bundle
2. 6 separate bundles with one blob each - this might be slightly better and we only lazily initialize the `TEST_BLOBS_BUNDLE_BYTES` when we need the blob, and only decode what we need, but then I haven't really compare the cost of decoding multiple small bytes object vs. decoding 1 single large bytes object.

But wasn't sure if there's any benefit of doing this and if it's worth bloating up our compilation and decode time, as we should be indifferent about the content of the blobs, as long as they are valid - similar to how we reuse one single static transaction in `static_valid_tx`. It won't increase the lighthouse binary size, I think it's just compilation time and memory usage (and `rust-analyzer` performance, which I've been struggling a lot with recently 😭 )

Reply:
> Do any of our tests assume different blobs within a block? 
I can't think of any scenario where this test would be useful (but I could be missing something), as we don't really care about content of blobs as long as they're valid, the resulting blob sidecars would be different for each blob anyway, and I think that's what we'd care about?

Reply:
I agree, just one bundle seems good enough.